### ISSUE TITLE
Fix focus style on search button

### ISSUE THREAD

**Issue by smcmurtry** [veteransaffairscanada/vac-benefits-directory#1410]
Where the button is clickable it should be the yellow focus style, where it is not clickable it should not have a focus style.
